install_config/install/docker_registry.adoc

```diff
@@ -169,9 +169,41 @@ $ oadm registry --service-account=registry \
 ----
 endif::[]
 
-[[viewing-logs]]
 
+[[registry-ip-address]]
+=== Registry IP Address
+
+Because the integrated registry is referenced by its service IP address,
+if you decide to delete and recreate the docker-registry service,
+you should either take care to re-use its address,
+or reboot the cluster after creating the new docker-registry service.
+
+[[re-using-the-address]]
+==== Re-using the Address
```

Comment:
Yeah, i had a feeling this was too deep (and the volume in each sub-sub-sub section too little to warrant it).  Will rework to use a labelled list.